src/cloudant/database.py

```diff
@@ -152,15 +152,12 @@ def create_document(self, data, throw_on_exists=False):
             corresponding to the new document in the database.
         """
         docid = data.get('_id', None)
-        doc = None
         if docid and docid.startswith('_design/'):
             doc = DesignDocument(self, docid)
         else:
             doc = Document(self, docid)
-        if throw_on_exists and doc.exists():
-            raise CloudantDatabaseException(409, docid)
         doc.update(data)
-        doc.create()
+        doc.create(throw_on_exists=throw_on_exists)
         super(CouchDatabase, self).__setitem__(doc['_id'], doc)
```

Comment:
Noting that `throw_on_exists=False` could result in your local database dict not being kept in sync with the server.